### ISSUE TITLE
Error: Duplicate attachMathJax call

### ISSUE THREAD

**Issue by SolangeUG** [atom-community/markdown-preview-plus#354]
[Enter steps to reproduce:]

1. I modified **atom stylesheet** by adding the following lines. The markdown styles updated correctly, and I was able to continue editing my document flawlessly, save it and close the editor.
````CSS
div.update-preview {
  font-size: 13px;
  font-family: "Segoe UI";
  blockquote {
    font-size: 13px;
    font-style: italic;
    color: #000;
  }
}
````
2. When I reopened the editor, a **popup with this error** opened and offered to create this issue. I should mention that the following is what my entire **styles.less** file looks like. I added the styles above when I noticed that the previous styles weren't "working anymore".
````CSS
.markdown-preview.markdown-preview,
atom-text-editor.editor .line .source-gfm {
  font-size: 13px;
  font-family: "Segoe UI";
  blockquote {
    font-size: 13px;
    font-style: italic;
    color: #000;
  }
}

div.update-preview {
  font-size: 13px;
  font-family: "Segoe UI";
  blockquote {
    font-size: 13px;
    font-style: italic;
    color: #000;
  }
}
````

**Atom**: 1.24.0 x64
**Electron**: 1.6.16
**OS**: Microsoft Windows 7 Professional 
**Thrown From**: [markdown-preview-plus](https://github.com/atom-community/markdown-preview-plus) package 2.5.4

### Stack Trace

Error: Duplicate attachMathJax call

```
At Duplicate attachMathJax call

Error: Duplicate attachMathJax call
    at attachMathJax (/packages/markdown-preview-plus/lib/mathjax-helper.js:66:11)
    at loadMathJax (/packages/markdown-preview-plus/lib/mathjax-helper.js:55:20)
    at Object.mathProcessor (/packages/markdown-preview-plus/lib/mathjax-helper.js:15:9)
    at UpdatePreview.update (/packages/markdown-preview-plus/lib/update-preview.js:35:27)
    at renderer.toDOMFragment (/packages/markdown-preview-plus/lib/markdown-preview-view.js:280:40)
    at /packages/markdown-preview-plus/lib/renderer.js:20:16
    at callbackFunction (/packages/markdown-preview-plus/lib/renderer.js:52:16)
```

### Commands

```
```

### Non-Core Packages

```
atom-ide-ui 0.9.1 
atom-ternjs 0.18.3 
language-matlab-octave 0.2.1 
language-scala 1.1.9 
language-sml 0.7.0 
markdown-preview-plus 2.5.4 
mathjax-wrapper 0.1.0 
```

Thanks in advance for you help! :smiley:

**Comment by lierdakil** [comment]
This error isn't in any way related to your stylesheet... And to be honest, this error was deliberately put in there as "something impossible happened" more or less. So I'm *really* not sure how this came to be. At least I couldn't reproduce it based on your description =\

Anyway, I'll rework the code a bit to avoid this. Hopefully, that works.

**Comment by lierdakil** [comment]
Let me know if you experience any weirdness on v2.5.5

**Comment by SolangeUG** [comment]
Hi @lierdakil :smiley: 

Thank you so much for your quick reply! I had never experienced any issue with **markdown-preview-plus** until I updated to version 2.5.4 and made those changes to the stylesheet. Hence, I can't really be absolutely sure that the issue was related to those changes.

I've just updated to v2.5.5 and so far so good! I'll sure let you know if any *surprises* arise! Thank you so much for the fix! That was fantastically quick! :smile:
